frontend/src/components/Editor.js

```diff
@@ -5,6 +5,16 @@ import "ace-builds/src-noconflict/mode-csharp";
 import "ace-builds/src-noconflict/theme-monokai";
 import { useFileUpload } from "./fileUploadContext";
 
+function searchFileName(fileId, files) {
+  const file = Object.keys(files).find(
+    (fileName) => files[fileName].fileId === fileId
+  );
+  if (file === undefined || file === null) {
```

Comment:
Maybe we can do `return file ?? "main.c";` instead of all this. 

Here is some more info on the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)

Reply:
Got it